src/families/filecoin/test-dataset.ts

```diff
@@ -0,0 +1,219 @@
+            index: 0,
+            freshAddress: SEED_IDENTIFIER,
+            freshAddressPath: "44'/461'/0'/0/0",
+            freshAddresses: [],
+            blockHeight: 0,
+            operations: [],
+            pendingOperations: [],
+            currencyId: "filecoin",
+            unitMagnitude: 18,
+            lastSyncDate: "",
+            balance: "100000000000",
+          },
+          transactions: [
+            {
+              name: "recipient and sender must not be the same",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 100,
+                gasPremium: "200",
+                recipient: SEED_IDENTIFIER,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                amount: new BigNumber("100000000"),
+                errors: {
+                  recipient: new InvalidAddressBecauseDestinationIsAlsoSource(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Not a valid address",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 100,
+                gasPremium: "200",
+                recipient: "novalidaddress",
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  recipient: new InvalidAddress(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Fee Required (Gas Premium)",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "1000",
+                gasLimit: 100,
+                gasPremium: "0",
+                recipient: ACCOUNT_1,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  gas: new FeeNotLoaded(),
+                },
+                warnings: {},
+              },
+            },
+            {
+              name: "Fee Required (Gas Fee Cap)",
+              transaction: fromTransactionRaw({
+                family: "filecoin",
+                method: 1,
+                version: 1,
+                nonce: 100,
+                gasFeeCap: "0",
+                gasLimit: 100,
+                gasPremium: "10000",
+                recipient: ACCOUNT_1,
+                amount: "100000000",
+              }),
+              expectedStatus: {
+                errors: {
+                  gas: new FeeNotLoaded(),
```

Comment:
it seems you expected "FeeNotLoaded" error in this case? it seems the CI sees there are no error in this case.